common/tests/s3_util_test.cpp

```diff
@@ -140,6 +140,24 @@ TEST(S3UtilTest, DirectIOTest) {
   fs::remove(file_path);
 }
 
+TEST(S3UtilTest, CallDestructorOnlyOnce) {
+  SDKOptions options;
+  Aws::InitAPI(options);
```

Comment:
Why call InitAPI()?
We can just remove the whole test, as it doesn't really test much but adds unneeded complexity to the S3Util interface.

Reply:
Good cache. Forgot to delete this. :)

Reply:
Changed and committed the code. 
I will leave the test here for now. Will delete when we all agree that it is not necessary. 

I feel that with this PR, the code https://github.com/pinterest/rocksplicator/blob/master/rocksdb_admin/admin_handler.cpp#L679-L712  will not be necessary anymore to avoid Aws::InitAPI() and Aps::ShutdownApi() not match problem.